tests/biblio-tests.el

```diff
@@ -103,6 +103,11 @@ month={Apr}, pages={147–156}}")
      (container . ["Lecture Notes in Computer Science" "Distributed Computing"])
      (references "10.1007/978-3-642-24100-0_10")
      (type . "book-chapter") (url . "https://dx.doi.org/10.1007/978-3-642-24100-0_10"))
+    ((backend . biblio-iacr-backend)
+     (title . "Universally Composable Security: A New Paradigm for Cryptographic Protocols")
+     (authors "Ran Canetti")
+     (containter . "Cryptology ePrint Archive, Report 2000/067") (type . "eprint")
+     (url . "https://eprint.iacr.org/2000/067") (doi . nil))
```

Comment:
I ran into a weird bug with the test suite, where if this sample item was inserted after the dblp samples, some tests in the -get-url section would fail. I don't understand buttercup well enough to debug this though